### PR TITLE
feat: embed timestamps

### PR DESCRIPTION
**Pull request by MemencioPerez**
I did this because I needed that feature, let me know if anything can be improved (I'm not very skilled).

### CODE REVIEW

File: src/main/java/me/micartey/webhookly/DiscordWebhook.java [hunk 1]
```diff
@@ -55,6 +55,8 @@ public void execute() throws IOException {
                     jsonEmbed.put("color", rgb);
                 });
 
+                jsonEmbed.put("timestamp", embed.getTimestamp());
```
Comment (micartey) [inline src/main/java/me/micartey/webhookly/DiscordWebhook.java]: Please wrap this statement in `Optional#ofNullable`

File: src/main/java/me/micartey/webhookly/embeds/EmbedObject.java [hunk 2]
```diff
@@ -17,6 +21,7 @@ public class EmbedObject {
     private String description;
     private String url;
     private Color color;
+    private String timestamp;
```
Comment (micartey) [inline src/main/java/me/micartey/webhookly/embeds/EmbedObject.java]: Please consider using one of the following options:
1. Use `OffsetDataType`
2. Use `long` (milliseconds)

I, personally, prefer the 2nd option

File: src/main/java/me/micartey/webhookly/embeds/EmbedObject.java [hunk 3]
```diff
@@ -25,4 +30,20 @@ public class EmbedObject {
...
+            timestamp = OffsetDateTime.ofInstant((Instant) temporalAccessor, Clock.systemDefaultZone().getZone()).toString();
+        } else {
+            timestamp = OffsetDateTime.from(temporalAccessor).toString();
+        }
+    }
+
+    public void setTimestampFromTimeInMillis(long timeInMillis) {
+        setTimestampFromTemporalAccessor(Instant.ofEpochMilli(timeInMillis));
+    }
+
```
Comment (micartey) [inline src/main/java/me/micartey/webhookly/embeds/EmbedObject.java]: Preferably, it shouldn't be required to add these methods.
Use the builder instead.

**Review by micartey — CHANGES_REQUESTED** [review]

**Comment by micartey** [comment]
> I did this because I needed that feature, let me know if anything can be improved (I'm not very skilled).

No worries - We are all here to learn. 

I created a review and requested some changes.  If you need help with anything, I can also make some of these changes or explain in detail.

File: src/main/java/me/micartey/webhookly/embeds/EmbedObject.java (hunk 3 above)
Comment (MemencioPerez) [reply src/main/java/me/micartey/webhookly/embeds/EmbedObject.java]: All right, I will remove them.

File: src/main/java/me/micartey/webhookly/embeds/EmbedObject.java (hunk 2 above)
Comment (MemencioPerez) [reply src/main/java/me/micartey/webhookly/embeds/EmbedObject.java]: I think OffsetDataTime is the best, as it returns a timestamp in ISO-8601 format when passed to String.

File: src/main/java/me/micartey/webhookly/DiscordWebhook.java (hunk 1 above)
Comment (MemencioPerez) [reply src/main/java/me/micartey/webhookly/DiscordWebhook.java]: Done.

File: src/main/java/me/micartey/webhookly/embeds/EmbedObject.java (hunk 2 above)
Comment (MemencioPerez) [reply src/main/java/me/micartey/webhookly/embeds/EmbedObject.java]: I changed the variable type to OffsetDateTime for now.

**Pull request merged**